src/index.html

```diff
@@ -6,13 +6,74 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
-    <title>Product cards</title>
+    <title>Product product-cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <section
+      class="product-card"
+      data-qa="card"
+    >
+      <img
+        class="product-card__product-image"
+        src="./images/imac.jpeg"
+        alt="apple-imac-27"
+      />
+
+      <div class="product-card__product-details">
+        <h3 class="product-card__product-details-title">
```

Comment:
Element with the class 'product-card__product-details-title' should be a 'div' or 'span' since it's not the main title of the page. Use 'h3' for subsections titles within the content of 'section' if needed.